editor/src/components/canvas/canvas-strategies/strategies/rearrange-grid-swap-strategy.ts

```diff
@@ -121,7 +121,7 @@ export const rearrangeGridSwapStrategy: CanvasStrategyFactory = (
         return emptyStrategyApplicationResult
       }
 
-      return strategyApplicationResult(commands)
+      return strategyApplicationResult(commands, 'rerender-all-elements')
```

Comment:
possibly one to fix here too

Reply:
Added comment about follow up in 38ec355d5.